osmoutils/accum/prefix.go

```diff
@@ -8,23 +8,16 @@ const (
 	positionPrefix    = "pos"
 )
 
-// formatAccumPrefix returns the key prefix used for any
-// accum module values to be stored in the KVStore.
-// Returns "accum/{key}" as bytes.
-func formatModulePrefixKey(key string) []byte {
-	return []byte(fmt.Sprintf("%s/%s", modulePrefix, key))
-}
-
 // formatAccumPrefix returns the key prefix used
 // specifically for accumulator values in the KVStore.
 // Returns "accum/acc/{name}" as bytes.
 func formatAccumPrefixKey(name string) []byte {
-	return formatModulePrefixKey(fmt.Sprintf("%s/%s", accumulatorPrefix, name))
+	return []byte(fmt.Sprintf(modulePrefix+"/"+accumulatorPrefix+"/%s", name))
 }
 
 // FormatPositionPrefixKey returns the key prefix used
 // specifically for position values in the KVStore.
-// Returns "accum/pos/{accumName}/{name}" as bytes.
+// Returns "accum/acc/pos/{accumName}/{name}" as bytes.
```

Comment:
notice that the old code actually would add `acc` here, making the old comment wrong :)